src/core/datastore/cachestore.spec.js

```diff
@@ -111,130 +111,206 @@ describe('CacheStore', () => {
         });
     });
 
-    it('should return the entities', (done) => {
+    it('should return the entities');
```

Comment:
It's very difficult to read the diff when tests are being changed this way. The best way to mark it as pending is to call `skip`.

Like so
`it.skip('should return the entities', () => {...})`.

The lines below are "out of sync" - they don't appear next to where they were changed, due to identical lines in different tests. So the entire diff is pretty hard to read. Doing it as mentioned above would make things much better.

Reply:
Ok I wasn't aware of this options.